api/src/main/resources/workbench.yaml

```diff
@@ -960,6 +960,48 @@ paths:
           description: A list of resources order by last access time desc order
           schema:
             $ref: "#/definitions/RecentResourceResponse"
+
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/user-recent-resource/updateEntry:
```

Comment:
I don't think you need the "entry" part, but if you decide to leave it, should be hyphenated - otherwise you're mixing camelCase and hyphenation in the same URL